src/test/resources/refaster/ShouldAddImportsRecipes.java

```diff
@@ -210,4 +212,59 @@ public J visitMethodInvocation(J.MethodInvocation elem, ExecutionContext ctx) {
         }
     }
 
+    /**
+     * OpenRewrite recipe created for Refaster template {@code ShouldAddImports.FileExists}.
+     */
+    @SuppressWarnings("all")
+    @NonNullApi
+    public static class FileExistsRecipe extends Recipe {
+
+        /**
+         * Instantiates a new instance.
+         */
+        public FileExistsRecipe() {}
+
+        @Override
+        public String getDisplayName() {
+            return "Refaster template `ShouldAddImports.FileExists`";
+        }
+
+        @Override
+        public String getDescription() {
+            return "Recipe created for the following Refaster template:\n```java\npublic static class FileExists {\n    \n    @BeforeTemplate()\n    boolean before(Path path) {\n        return path.toFile().exists();\n    }\n    \n    @AfterTemplate()\n    @UseImportPolicy(value = ImportPolicy.STATIC_IMPORT_ALWAYS)\n    boolean after(Path path) {\n        return exists(path);\n    }\n}\n```\n.";
+        }
+
+        @Override
+        public TreeVisitor<?, ExecutionContext> getVisitor() {
+            JavaVisitor<ExecutionContext> javaVisitor = new AbstractRefasterJavaVisitor() {
+                final JavaTemplate before = Semantics.expression(this, "before", (java.nio.file.Path path) -> path.toFile().exists()).build();
+                final JavaTemplate after = Semantics.expression(this, "after", (java.nio.file.Path path) -> exists(path)).build();
+
+                @Override
+                public J visitMethodInvocation(J.MethodInvocation elem, ExecutionContext ctx) {
+                    JavaTemplate.Matcher matcher;
+                    if ((matcher = before.matcher(getCursor())).find()) {
+                        maybeAddImport("java.nio.file.Files", "exists");
```

Comment:
Not that this `maybeAddImport` is not necessary when we use `Files.exists(path)` in the `@AfterTemplate` instead of a static import there. But unfortunately in error-prone-support they use this style often, [such as in MockitoRules](https://github.com/PicnicSupermarket/error-prone-support/blob/967a44690931064be5810595e84598b11095d067/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/MockitoRules.java#L26-L36).